### PR TITLE
feat(ifo): List ESPL IFO

### PR DESCRIPTION
**Pull request by Chef-Yogi**
- [x] IFO contract address
- [x] Token Image
- [x] Header Image

### CODE REVIEW

File: apps/web/src/config/constants/ifo.ts [hunk 1]
```diff
@@ -6,6 +6,31 @@ import { Ifo } from './types'
 export const cakeBnbLpToken = new ERC20Token(ChainId.BSC, CAKE_BNB_LP_MAINNET, 18, 'CAKE-BNB LP')
 
 const ifos: Ifo[] = [
+  {
+    id: 'espl',
+    address: '0x57eb2e676863435a140f8bc2a0f0e955c36ed446', // TODO wait for operation SC
+    isActive: true,
+    cIFO: false,
```
Comment (ChefMomota) [inline apps/web/src/config/constants/ifo.ts:13]: remove

File: apps/web/src/config/constants/campaigns.ts [hunk 2]
```diff
@@ -407,6 +407,12 @@ const campaigns: Campaign[] = [
     title: 'SQUAD',
     badge: 'ifo-squad.svg',
   },
+  {
+    id: '512100000',
+    type: 'ifo',
+    title: 'ESPL Arena',
```
Comment (ChefMomota) [inline apps/web/src/config/constants/campaigns.ts:413]: ESPL

File: apps/web/src/config/constants/ifo.ts [hunk 3]
```diff
@@ -6,6 +6,31 @@ import { Ifo } from './types'
...
+    name: 'Esports Arena',
```
Comment (ChefMomota) [inline apps/web/src/config/constants/ifo.ts:14]: ESPL

**Comment by ChefMomota** [comment]
Symbol should be `ARENA`

**Review by chef-jojo — DISMISSED** [review]

**Review by chefilip — DISMISSED** [review]

**Comment by ChefMomota** [comment]
`espl-bg.png` should be `arena-bg.png`

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**